contrib/dev/Vagrantfile

```diff
@@ -173,7 +173,8 @@ Vagrant.configure(2) do |config|
       dev.vm.provision :ansible_local do |ansible|
         ansible_extra_vars = {
           "skydive_extra_config" => $skydive_extra_config,
-          "skydive_fabric_default_interface" => "eth0"
+          "skydive_fabric_default_interface" => "eth0",
+          "skydive_release" => ENV.fetch("SKYDIVE_RELEASE", "v0.17.0"),
```

Comment:
There is already a default value for the skydive_release  in the ansible roles. I would prefer to specify `skydive_release` only if the environment variable is defined